soco/core.py

```diff
@@ -432,20 +432,43 @@ def ramp_to_volume(self, volume, ramp_type='SLEEP_TIMER_RAMP_TYPE'):
         return int(response['RampTime'])
 
     @only_on_master
-    def play_from_queue(self, index, start=True):
-        """ Play a track from the queue by index. The index number is
-        required as an argument, where the first index is 0.
+    def jump_to_track(self, index):
+        """ Jump to another track in the queue.
```

Comment:
No space after """ and include a blank line after the first single line and the larger explanation below